### DISCUSSION TITLE
Chapter title filter AND/OR every chapter after index N

### DISCUSSION THREAD

**Discussion by God-damnit-all** [lncrawl/lightnovel-crawler#2482] [Q&A]
Using the CLI tool, I want to download every chapter in which the title starts with, for example, "Foobar -", I haven't been able to find any CLI option that allows me to do that.

Failing that, I want to download after chapter after index 15. There is a `--range` option, but it requires you list a `TO` argument, when what I want is for it to select everything after that index without having to know the final index.

I'm guessing the title filter is only doable via a Python script at the moment, but is downloading every chapter after index N possible?

**Comment by dipu-bd** [comment]
it's not possible now. I've to update the tool. just put a very big value for now
